src/averageAges.js

```diff
@@ -37,7 +40,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter((woman) => (
+    woman.sex === 'f'
+    && (
+      withChildren
+        ? people.some(person => woman.name === person.mother)
+        : true
+    )
+  ));
+
+  return women
+    .reduce((count, woman) =>
+      count += (woman.died - woman.born), 0)
```

Comment:
If you remove the assignment operator you will also NOT get the following error
`Assignment to function parameter 'count'`